rule.go

```diff
@@ -58,6 +58,8 @@ type Rule struct {
 	Vars map[string]*Var
 	// Metas is a slice of Metadata.
 	Metas Metadatas
+	// Flowbits is a slice of Flowbit.
```

Comment:
please pull from head and handle merge issues.
And please update with Stringers for the new type.